konig-core/src/main/java/io/konig/core/vocab/Konig.java

```diff
@@ -37,6 +37,7 @@
 	public static final String INPUT_SHAPE_OF = "http://www.konig.io/ns/core/inputShapeOf";
 	public static final String IRI_TEMPLATE = "http://www.konig.io/ns/core/iriTemplate";
 	public static final String IRI_FORMULA = "http://www.konig.io/ns/core/iriFormula";
+	public static final String ONE_OF = "http://www.konig.io/ns/core/OneOf";
```

Comment:
We should not be minting a term named `konig:OneOf`.
The "One Of" label is just a heading for a spreadsheet.

The list of values should always be accessed through the `sh:or` property.